app/vep/models/pipeline_model.py

```diff
@@ -55,11 +48,13 @@ class ConfigIniParams(BaseModel):
     biotype: bool = False
     transcript_version: int = 1
     canonical: int = 1
-    gff: str = "/nfs/public/rw/enswbsites/dev/vep/test/genes.gff3.bgz"
+    sort: int = 1
+    # TODO: use genome_id from client payload to set gff+fasta paths
```

Comment:
We have already discussed these hardcoded parts, Are these comments necessary? is there an actual update here that I'm missing?

Reply:
Just a reminder that dynamic paths need to be implemented for beta-2 (i.e. in this sprint?). Will turn it to tickets and remove comment.

Reply:
> dynamic paths need to be implemented for beta-2

Erm, isn't beta2 happening earlier/independently of VEP?

> (i.e. in this sprint?)

We would need loaded production data, an up-to-date grpc service probably pointing at the updated database, plus updates to the web-metadata-api that will pick the new attributes from the grpc response. Sounds unlikely for this sprint.

Reply:
Yes, you're right, slow morning. We still need the tickets in the backlog if not already there.